### PR TITLE
Give fork mode more exit time, otherwise runs into SIGTERM.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/bot/fuzzers/libfuzzer.py [hunk 1]
```diff
@@ -46,6 +46,9 @@ class LibFuzzerCommon(object):
   # Window of time for libFuzzer to exit gracefully before we KILL it.
   LIBFUZZER_CLEAN_EXIT_TIME = 10.0
 
+  # Additional window of time for libFuzzer fork mode to exit gracefully.
+  LIBFUZZER_FORK_MODE_CLEAN_EXIT_TIME = 50.0
```
Comment (Dor1s) [inline src/python/bot/fuzzers/libfuzzer.py]: Are you sure 50 is enough? Maybe 60-100? I think we should fix it in libFuzzer (if possible) and then get rid of this, so using 100 for extra safety sounds good to me.

Comment (Dor1s) [reply src/python/bot/fuzzers/libfuzzer.py]: for instance, I had a 60 seconds overhead locally

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: Done!

**Comment by oliverchang** [comment]
LGTM

**Pull request merged**